### PR TITLE
Add CMakeLists.txt in root folder.

### PR DESCRIPTION
**Pull request by ActoryOu**
*Issue #, if available:*
NA

*Description of changes:*
Add CMakeLists.txt in root folder. Then WebRTC SDK-C can clone this repo by ExternalProject_Add.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.6.3)
```
Comment (amazonKamath) [inline CMakeLists.txt:1]: is this the cmake version used by KVS team?

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+project(kvssdp C)
```
Comment (amazonKamath) [inline CMakeLists.txt:2]: Do we need to set C standard?

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+include(sdpFilePaths.cmake)
+
+include_directories(${SDP_INCLUDE_PUBLIC_DIRS})
```
Comment (amazonKamath) [inline CMakeLists.txt:6]: I am not sure, but curious why not use target_include_directories

**Review by amazonKamath — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (ActoryOu) [reply CMakeLists.txt:1]: Yes, this version is same as [WebRTC-C-SDK one](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/CMakeLists.txt#L1).

File: CMakeLists.txt (hunk 2 above)
Comment (ActoryOu) [reply CMakeLists.txt]: I just removed this language to use default setting (C/CXX).
I can still pass test cases with that.

File: CMakeLists.txt (hunk 3 above)
Comment (ActoryOu) [reply CMakeLists.txt]: Update to target_include_directories.

**Review by hassanctech — APPROVED** [review]
lgtm

**Pull request merged**